entrypoint.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import pwd
+import subprocess
+from dotenv import load_dotenv
+
+
+MY_DIR = os.path.abspath(os.path.dirname(__file__))
+HOST_UID = os.stat(MY_DIR).st_uid
+HOST_USER = 'autoapi_user'
+DOTENV_PATH = os.path.join(os.path.dirname(__file__), '.env')
+
+
+def entrypoint(argv):
+    '''
+    This is a Docker entrypoint that configures the container to run
+    as the same uid of the user on the host container, rather than
+    the Docker default of root. Aside from following security best
+    practices, this makes it so that any files created by the Docker
+    container are also owned by the same user on the host system.
+    '''
+
+    if HOST_UID != os.geteuid():
+        if not does_uid_exist(HOST_UID):
+            username = HOST_USER
+            while does_username_exist(username):
+                username += '0'
+            home_dir = '/home/%s' % username
+            subprocess.check_call([
+                'adduser',
+                '-h', home_dir,
+                '-u', str(HOST_UID),
+                '-S', username,
+            ])
+        os.environ['HOME'] = '/home/%s' % pwd.getpwuid(HOST_UID).pw_name
+        os.setuid(HOST_UID)
+
+    if not os.path.exists('/autoapi/node_modules'):
+        subprocess.check_call(
+            "ln -s /tmp/node_modules /autoapi/node_modules",
+            shell=True
+        )
```

Comment:
As mentioned [above](https://github.com/18F/autoapi/pull/59/files#r68235251), this is kind of janky.  I'd rather copy the files but that appears to be difficult due to permissions+alpine, and running `npm install` in the entrypoint every single time slows things down too much, so I opted for a symlink. Bleh.